src/pages/Controllers/ControllerChart/ControllerChart.js

```diff
@@ -25,19 +25,19 @@ export default function ControllerChart({ chartData, totalLabel }) {
             className="p-list__item p-legend__item is-blocked"
             data-test="legend-blocked"
           >
-            Blocked: {chartData.blocked}
+            Blocked: {chartData.blocked || 0}
```

Comment:
The data structure of `chartData` is quite strict, requiring that it has `blocked`, `alert`, and `running`. There are now two places in this file and in others making sure this data has a default value of 0 defined. 

I feel like you should update this to not take `chartData` but to take `blockedCount`, `alertCount`, and `runningCount` all of which have defaults of `0`.

Reply:
Seems sensible but it's beyond the scope of this PR - can you create a separate ticket please?

Reply:
https://github.com/canonical-web-and-design/jaas-dashboard/issues/567